R-package/README.md

```diff
@@ -0,0 +1,564 @@
+
+sudo pip install -U scikit-learn
+```
+
+<br>
+
+Then, download the *rgf-python* package and install it using the following commands,
+
+<br>
+
+
+```R
+
+git clone https://github.com/RGF-team/rgf.git
+
+cd rgf/python-package
+
+sudo python setup.py install
+
+```
+
+<br>
+
+*FastRGF* will be installed successfully only if gcc >= 5.0.
+
+<br><br>
+
+
+
+#### **Macintosh OSX**            [ Python >= 3.4 ]
+
+<br>
+
+Upgrade python to version 3 using, 
+
+
+```R
+
+brew upgrade python
+
+```
+<br>
+
+Then install the dependencies for *RGF* and *FastRGF*
+
+<br>
+
+```R
+sudo pip3 install --upgrade setuptools
+
+sudo pip3 install -U numpy
+
+sudo pip3 install --upgrade scipy
+
+sudo pip3 install -U scikit-learn
+```
+<br>
+
+
+The *FastRGF* module requires a gcc >= 8.0. To install *gcc-8* (or the most recent gcc) with brew follow the next steps,
+
+<br>
+
+```R
+
+# before running the following commands make sure that the most recent Apple command line tools for Xcode are installed
+
+brew update
+
+brew upgrade
+
+brew info gcc
+
+brew install gcc
+
+brew cleanup
+
+```
+<br>
+
+After the newest gcc version is installed the user should navigate to */usr/local/bin* and if a *gcc* file exists (symbolic link) then the user should delete it. Then the user should
+run the following command,
+
+<br>
+
+```R
+
+sudo ln -s /usr/local/bin/gcc-8 /usr/local/bin/gcc
+
+```
+
+<br>
+
+The user should then verify that the gcc has been updated using,
+
+<br>
+
+```R
+
+gcc -v
+
+which gcc
+
+```
+
+<br>
+
+After the new gcc is installed the user should continue with the installation of *rgf-python*,
+
+<br>
+
+```R
+
+git clone https://github.com/RGF-team/rgf.git
+
+cd /rgf/RGF/build
+
+export CXX=/usr/local/bin/g++-8 && export CC=/usr/local/bin/gcc-8
+
+cmake /rgf/RGF /rgf/FastRGF
+
+make
+
+sudo make install
+
+cd /rgf/python-package
+
+sudo python3 setup.py install
+
+```
+<br>
+
+After a successful rgf-python installation the user should open an R session and give the following *reticulate* command to change to the relevant (brew-python) directory (otherwise the RGF package won't work properly),
+
+<br>
+
+
+```R
+
+reticulate::use_python('/usr/local/bin/python3')
+
+
+```
+
+<br>
+
+and then,
+
+<br>
+
+
+```R
+
+reticulate::py_discover_config()
+
+
+```
+
+<br>
+
+to validate that a user is in the python version where *RGF* or *FastRGF* are installed. Then,
+
+<br>
+
+```R
+
+install.packages(RGF)
+
+
+library(RGF)
+
+
+```
+
+<br>
+
+to load the R package. It is possible that the following warning in the R session appears if *FastRGF* is not installed,
+
+<br>
+
+```R
+
+UserWarning: Cannot find FastRGF executable files. FastRGF estimators will be unavailable for usage.
+  warnings.warn("Cannot find FastRGF executable files. FastRGF estimators will be unavailable for usage.")
+  
+```
+
+<br><br>
+
+#### **Windows OS**            [ Python >= 3.4 ]
+
+<br>
+
+**NOTE : CURRENTLY THE PACKAGE ON WINDOWS CAN BE USED ONLY FROM THE COMMAND LINE (cmd)**
```

Comment:
Why?

Reply:
The rgf_python package once installed on Windows can be used from an IDE or GUI. However, I wasn't able to make the RGF R package to work from within an IDE (such as Rstudio) or the R GUI.
I tried many things, such as running as administrator or with a different python version, however it was impossible. Probably I should open an issue with a help wanted badge.

Reply:
Got it! I assumed that maybe it's some known reticulate restriction or anything else...